src/model/RootModel.js

```diff
@@ -131,6 +131,17 @@ export default class RootModel extends Model {
 		return remapped;
 	}
 
+	mark () {
+		const keys = Object.keys( this.mappings );
+
+		let i = keys.length;
+		while ( i-- ) {
+			this.mappings[ keys[i] ].mark();
+		}
```

Comment:
Fair enough :smile:. For some reason I default to performance mode, which is definitely premature optimization here.